setup.py

```diff
@@ -1,9 +1,11 @@
 from setuptools import setup, find_packages
 
-version = '1.5.dev0'
+
+from tzlocal import __version__
```

Comment:
Importing the local package from setup.py is a bad idea for multiple reasons.

Reply:
That's why the `__init__` now doesn't import anything at module load time... this is tested many places... makes it easier to maintain the version. The purpose is that you now have `__version__` in `__init__`

Reply:
See the comment on that code in `__init__`